### DISCUSSION TITLE
fieldSetAnatomy Missing from @chakra-ui/react/anatomy

### DISCUSSION THREAD

**Discussion by oka4a** [chakra-ui/chakra-ui#9120] [General] [answered]
Thank you so much for this fantastic package and for the exciting new features in version 3. Chakra UI has been incredibly helpful in our projects!

The `fieldSetAnatomy `export is currently missing from `@chakra-ui/react/anatomy`. I’d like to use it to override the fieldset recipe for custom styling.

For example, I’m currently able to do the following with other components:
```
import { defineSlotRecipe } from "@chakra-ui/react";
import { tableAnatomy } from "@chakra-ui/react/anatomy";

const tableRecipe = defineSlotRecipe({
  slots: tableAnatomy.keys(),
  base: {
    body: {
      _empty: {
        position: "relative",
        h: "72px",
        _after: {
          position: "absolute",
          content: '"No data available in table"',
          transform: "translateX(-50%)",
          left: "50%",
          textAlign: "center",
          padding: "1.4rem",
          color: "gray.500",
          bg: "white",
          width: "100%",
          fontWeight: 600,
          fontSize: "1rem",
        },
      },
    },
  },
  })
```
**Questions**
- Is there a reason `fieldSetAnatomy` is not available in the anatomy module?
- Is there an alternative or recommended approach to achieve similar functionality for fieldsets?

Thank you for looking into this!

**Comment by segunadebayo** [comment, marked as answer]
Good catch. Will add this right away.

**Reply by segunadebayo** [reply]
This is now fixed.

**Reply by oka4a** [reply]
Thank you so much for the quick response and for fixing it! I really appreciate it.
